docs/manuals/source/Appendix/Debugging.rst

```diff
@@ -158,26 +158,60 @@ Installing debug symbols packages
 .. index::
    single: debug symbols package; dbg; debuginfo; debug
 
-Our Linux packages are stripped of debugging symbols, so you need an extra step to install their rpm debuginfo or deb dbg equivalent
+Our Linux packages are stripped of debugging symbols, so you need an extra step to install their rpm debuginfo or deb dbg equivalent.
 
+.. code-block:: shell-session
+   :caption: Installing bareos debug symbols package on deb system
 
-   .. code-block:: shell-session
-      :caption: Installing bareos debug symbols package on deb system
+   apt install bareos-dbg gdb
 
-      apt install bareos-dbg gdb
 
+.. code-block:: shell-session
+   :caption: Installing all Bareos debug symbols on (RH)EL system
 
-   .. code-block:: shell-session
-      :caption: Installing Bareos debug symbols on (RH)EL system
+   dnf debuginfo-install bareos-*-debuginfo
+
+
+.. code-block:: shell-session
+   :caption: Installing all Bareos debug symbols on (open)SUSE system
+
+   zypper --plus-content bareos-debuginfo --plus-content bareos-debugsource install bareos*debuginfo bareos*debugsource
+
+
+
+.. Note:: If you want to debug only a specific daemon, you have to install only its related **-debuginfo** packages. For example with |fd|
+
+.. code-block:: shell-session
+   :caption: Installing |fd| debug symbols package on deb system
 
-      dnf --enablerepo bareos-debuginfo install bareos-director-debuginfo
+   apt install bareos-dbg gdb
+
+
+.. code-block:: shell-session
+   :caption: Installing |fd| debug symbols on (RH)EL system
+
+   dnf debuginfo-install gdb bareos-debuginfo bareos-filedaemon-debuginfo bareos-filedaemon-python3-plugin-debuginfo bareos-common-debuginfo bareos-debugsource
```

Comment:
maybe we also want to use `bareos-filedaemon-*-debuginfo` here ?

Reply:
good catch, done